addons/mail/static/src/models/message.js

```diff
@@ -75,6 +75,12 @@ Model({
                 if ("module_icon" in data) {
                     originThreadData.moduleIcon = data.module_icon;
                 }
+                if ("user_follower_id" in data && data.user_follower_id && this.messaging.currentPartner) {
+                    originThreadData.followers = [{
```

Comment:
Why do we modify js in this task ? Just to display a notification ? Is that costly (aka what are the added queries, ...) ? As we recently add troubles in our prod db due to added notifications and checks in discuss, we try to keep this under control even if this means sometimes a bit less information :)